app/src/main/java/com/example/suberduberuber/Car.java

```diff
@@ -0,0 +1,38 @@
+package com.example.suberduberuber;
+
+public class Car {
+    private String make;
+    private String model;
+    private int year;
+    private String licensePlate;
+    private String color;
+
```

Comment:
Maybe it'd be handy to have a minimal constructor, with say just license plate as the identifier. Unless we want to always assert that drivers enter all the info, color, etc. through the driver registration process, or edit driver-edit profile UI.

Reply:
That's all I can think of, looks really good Robert!

Reply:
Good point - I figure we will likely always have that info required but it doesn't hurt to have it. Added it with default values set to all other attributes. Also added setters for the other attributes since they may be added down the line.